src/main.rs

```diff
@@ -145,7 +145,7 @@ fn run_app() -> std::io::Result<i32> {
         eprintln!(
             "\
     The main way to use delta is to configure it as the pager for git: \
-    see https://github.com/dandavison/delta#configuration. \
+    see https://github.com/dandavison/delta#get-started. \
```

Comment:
There is also an idea to link to [Configuration - delta](https://dandavison.github.io/delta/configuration.html), but it is probably a beginner like me who sees this message, so it is more natural to lead to <https://github.com/dandavison/delta#get-started>.